pyanaconda/installclasses/installclass_atomic.py

```diff
@@ -0,0 +1,120 @@
+#
+# installclass_atomic.py
+#
+# Atomic-specific partitioning defaults
+#
+# Copyright (C) 2014  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+import shutil
+from pyanaconda.installclasses.fedora import FedoraBaseInstallClass
+from pyanaconda.installclasses.fedora_server import FedoraServerInstallClass
+from pyanaconda.product import productVariant
+from pyanaconda.core import util
+
+import logging
+log = logging.getLogger("anaconda")
+
+__all__ = ['AtomicInstallClass']
+
+
+class AtomicInstallClass(FedoraBaseInstallClass):
+    name = "Atomic Host"
+    stylesheet = "/usr/share/anaconda/pixmaps/atomic/fedora-atomic.css"
+    sortPriority = 11000
+    hidden = False
+    defaultFS = "xfs"
+
+    if productVariant != "Atomic":
+        hidden = True
+
+    def __init__(self):
+        self.localemap = {}  # loaded lazily
+        FedoraBaseInstallClass.__init__(self)
+
+    def setDefaultPartitioning(self, storage):
+        FedoraServerInstallClass.createDefaultPartitioning(storage)
```

Comment:
Could this install class be based on Fedora Server install class?

Reply:
Again I'm not sure we want to do that. Yeah this one method is the same but what if FedoraServer will change something else in future.

Basically I don't think we want to mix hierarchy in this. If something we can use that mixin but these install classes are not extending one to other by principle.

Reply:
Ok. I don't like this solution, but we will change install classes anyway.